tox.ini

```diff
@@ -5,3 +5,4 @@ envlist = py{27,35}
 commands = py.test -v
 deps =
     pytest
+    -rrequirements.txt
```

Comment:
`-r requirements.txt`

Reply:
not a typo, but a syntax. https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints